src/components/questions/questionList/QuestionList.css

```diff
@@ -0,0 +1,9 @@
+.q_list_btns {
+  display: flex;
+  margin-left: 15px;
+}
+
+button {
+  padding: 5px;
+  margin-right: 20px;
+}
```

Comment:
be careful with very broad selectors like `button`, as they'll impact everything on the page, including buttons in other components/widgets.